app/views/HomePageView.scala.html

```diff
@@ -14,25 +14,27 @@
  * limitations under the License.
  *@
 
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.html.components._
+
 @this(
-        layout: templates.Layout,
-        govukNotificationBanner: GovukNotificationBanner
+layout: templates.Layout,
+govukPhaseBanner: GovukPhaseBanner,
+notificationBanner: notificationBanner,
+paragraphBody: paragraphBody
 )
 
-@(downloadReady: Boolean = false)(implicit request: Request[_], messages: Messages)
+@(downloadReady: Boolean)(implicit request: Request[_], messages: Messages)
 
 @layout(pageTitle = s"${messages("homepage.title")} - ${messages("site.govuk")}", showBackLink = false, headerShouldHaveServiceLink = false) {
 
     @if(downloadReady) {
-        @govukNotificationBanner(NotificationBanner(
-            title = Text(messages("successBanner.title")),
-            content = HtmlContent(f"""
-                <h3 class='govuk-notification-banner__heading'>
-                    ${messages("homepage.downloadBanner.content")}
-                    <a href="${routes.FileReadyController.onPageLoad().url}" class='govuk-link'>${messages("homepage.downloadBanner.linkText")}</a>.
-                </h3>"""),
-            bannerType = Some("success"),
-            role=Some("alert"))
+        @notificationBanner(
```

Comment:
Would we be able to still use the official govukNotificationBanner, but pass in the html, as this is the implementation on the component site
```
@import uk.gov.hmrc.govukfrontend.views.html.components._

@this(govukNotificationBanner : GovukNotificationBanner)

@()(implicit messages: Messages, request: RequestHeader)


@html = {
<p class="govuk-notification-banner__heading">
    You have 7 days left to send your application.
    <a class="govuk-notification-banner__link" href="#">View application</a>.
  </p>


}
@govukNotificationBanner(NotificationBanner(
    content = HtmlContent(html)
  ))
  
  ```
  You can have the new notificationBanner.scala.html be this part
  
  ```
  <p class="govuk-notification-banner__heading">
    You have 7 days left to send your application.
    <a class="govuk-notification-banner__link" href="#">View application</a>.
  </p>
  ```
  And take in a title and link to fill in the content

Reply:
Done.